etc/requirements.txt

```diff
@@ -62,3 +62,6 @@ sortedcontainers==1.4.4
 intervaltree==2.1.0
 
 cachetools==1.1.5
+
+# For financial risk calculations
+empyrical>=0.1.9
```

Comment:
Our setup.py should already convert this to a `>=`, so you can pip install zipline with a newer version of empyrical.